alembic/env.py

```diff
@@ -0,0 +1,68 @@
+from logging.config import fileConfig
+
+# from sqlalchemy import engine_from_config
+# from sqlalchemy import pool
+
+from alembic import context
+from src.database.session import engine_sync
+from src.database.models.models import City, Temperature
+from src.database.base import Base
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = Base.metadata
+
+# other values from the config, defined by the needs of env.py,
+# can be acquired:
+# my_important_option = config.get_main_option("my_important_option")
+# ... etc.
+
+
+def run_migrations_offline() -> None:
+    """Run migrations in 'offline' mode.
+
+    This configures the context with just a URL
+    and not an Engine, though an Engine is acceptable
+    here as well.  By skipping the Engine creation
+    we don't even need a DBAPI to be available.
+
+    Calls to context.execute() here emit the given string to the
+    script output.
+
+    """
+    connectable = engine_sync
```

Comment:
The `run_migrations_offline` function should not use `engine_sync` as it is intended to run without a database connection. Consider configuring the context with a URL instead of using a connectable object.